geniza/common/migrations/0001_data_editor_group.py

```diff
@@ -0,0 +1,67 @@
+# Generated by Django 3.1.6 on 2021-02-04 20:48
+
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+CONTENT_EDITOR = 'Content Editor'
+# permissions for content editor
+content_editor_perms = {
+    'corpus': [
+        'view_library'
+    ]
+}
+
+CONTENT_ADMIN = 'Content Admin'
+# additional permissions for content admin (also get content edit permissions)
+content_admin_perms = {
+    'corpus': [
+        'add_library', 'change_library', 'delete_library',
+    ]
+}
```

Comment:
I thought the other permission level was called "global admin" – is that something else? Did we change it?

Reply:
@thatbudakguy to me global admin == superuser

Since they are making some controlled vocabulary not editable by content editors I thought it might be useful to make a content admin group so they don't have to make everyone superusers. Maybe worth discussing when we request review/testing?

Reply:
ah, I see - so we don't need to define perms for global admin since it's an existing django category, but we do need another level between that and content editor. that explanation makes sense, and i think this naming makes sense too – but agree good to discuss in testing.